Sources/NIOCore/EventLoop.swift

```diff
@@ -14,6 +14,9 @@
 
 import NIOConcurrencyHelpers
 import Dispatch
+#if os(Linux)
```

Comment:
The `#if` guard should be unnecessary, `CNIOLinux` is just empty on Darwin.

Reply:
Happy to remove, but looking around the repo all existing imports of `CNIOLinux` _are_ guarded:

```
$  git grep -B2 "import CNIOLinux"
```